File: tests/expected/scope.cpp
#include <cstdint>   // NOLINT(build/include_order)
#include <iostream>  // NOLINT(build/include_order)
int global_var = 10;
inline void test_global() {
  global_var = 20;
  std::cout << global_var;
  std::cout << std::endl;
}

inline void show() { test_global(); }

int main(int argc, char** argv) { show(); }

File: tests/expected/scope.dart
// @dart=3.4
import 'package:sprintf/sprintf.dart';

int global_var = 10;
test_global() {
//global global_var
  global_var = 20;
  print(sprintf("%s", [global_var]));
}

show() {
  test_global();
}

main(List<String> argv) {
  show();
}
